### PR TITLE
Edit borrower

### PR DESCRIPTION
**Pull request by linyutinglyt**
editing borrower. parsers and its test to be implemented.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditBorrowerCommand.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        return new CommandResult(String.format(MESSAGE_EDIT_BORROWER_SUCCESS, editedBorrower));
+    }
+
+    /**
+     * Creates and returns a {@code Borrower} with the details of {@code borrowerToEdit}
+     * edited with {@code editBorrowerDescriptor}.
+     */
+    private static Borrower createEditedBorrower(Borrower borrowerToEdit,
+                                                 EditBorrowerDescriptor editBorrowerDescriptor) {
+        assert borrowerToEdit != null;
```
Comment (hoholyin) [inline src/main/java/seedu/address/logic/commands/EditBorrowerCommand.java]: maybe you can use `requireNonNull()` here

**Review by hoholyin — COMMENTED** [review]
👍

File: src/main/java/seedu/address/logic/commands/EditBorrowerCommand.java [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+ * Edits the details of an existing Borrower in the borrower record.
+ */
+public class EditBorrowerCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the borrower identified "
+            + "by borrower ID. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
```
Comment (shiyao821) [inline src/main/java/seedu/address/logic/commands/EditBorrowerCommand.java]: Is INDEX required?

**Review by hoholyin — APPROVED** [review]
LGTM!

**Pull request merged**